### ISSUE TITLE
2023.02.07.0-26 working like normal but still gives blink code

### ISSUE THREAD

**Issue by YoureNotDead** [NebraLtd/helium-syncrobit#16]
After flashing 2023.02.07.0-26, my Syncrobit hotspot is online and active on the explorer how ever there still is a flashing green light on the Syncrobit board, it flashes 4 times then stops and repeats. I had saw on one of the other issue posts that 4 flashes means no connection but mine is wired and working like normal. I imagine this is just a bug but I figured I would post it here to make it known.

**Comment by shawaj** [comment]
This isn't an issue that we can fix I don't think... As I think it's built into the balenaOS that we use from upstream.

Which board is this green light on though? The main motherboard or the Raspberry Pi CM4? Or somewhere else? 

If you can take a few pictures it would be helpful

**Comment by TheCryptoMoe** [comment]
Same here, I thought it was an SD Card issue, turns out its working fine? I luckily have two 32gb memory cards I been alternating with. I just reflashed it, the code is 4 green flashing lights... on repeat but seems to be witnessing just fine now.. :)

**Comment by YoureNotDead** [comment]
https://user-images.githubusercontent.com/47010638/229303807-e8a7e2dd-367b-4e16-920e-76f67562d2df.mp4

Ive also installed balenaOS to this syncrobit at one point and it seemed to use the green light as an activity light or on the webpage you were able to tell it to flash for a minute to identify the device. However it also seems the the raspberry pi will also default a 4 blink code for no bootable os found as I had a bad flash at one point and got no response other than that flashing light. Still working as normal so far!

Quick edit: This flash happens on both wired and wireless connection.

Edit2: Maybe it is due to using a wired connection. I had swapped my Syncrobit to use wifi last night and the flash code stayed but it just rebooted after taking that video and is still using wifi and now the flash code is gone.

**Comment by shawaj** [comment]
Ah ok so that's the light on the actual Syncrobit main board. Don't think we intentionally use that for anything so it's probably just accidental.

We will look into it though as might be able to make it more useful

**Comment by joaopaulo529** [comment]
@shawaj hello shawaj and hello everyone, how is the last update going?  I have many to flash but I can't flash install and uninstall all the time😔 I wanted to install and not move anymore.  I see nebra's tireless effort and thank you 👏👏 but I'm afraid that not everything is with the ota yet, and now comes the change to solana.. i'm thinking of waiting until the solana migration and change and see if by then everything is going well.. what do you think?  and now the syncrobit are doing the OTA and they are working?? greetings

**Comment by shawaj** [comment]
@joaopaulo529 the auto update is working fine now.

@YoureNotDead are you happy if we close this issue now?

**Comment by YoureNotDead** [comment]
@shawaj certainly! Like I had said at the start, I wasn't sure if this was a real issue or not so I just wanted to point it out.

**Comment by joaopaulo529** [comment]
hello 🙂 everything is great now. 👍👍👍

nebra top 😉🙏🙂🙂

complements

No dia 24/05/2023, às 22:24, YoureNotDead ***@***.***> escreveu:

﻿

@shawaj<https://github.com/shawaj> certainly! Like I had said at the start, I wasn't sure if this was a real issue or not so I just wanted to point it out.
